test/unit/instrumentation/http/outbound.test.js

```diff
@@ -340,7 +340,7 @@ tap.test('should add data from cat header to segment', (t) => {
       }
       return emit.apply(this, arguments)
     }
-    // This is really fucking gross.
+    // This is really freakin gross.
```

Comment:
lol. i mean you could just remove this. i don't think it provides actionable guidance :)

Reply:
lol true, one second

Reply:
removed the line